samples/core/kfp_env_validation/kfp_env_validation.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>Do we really need this after submitting for run?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/2843/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>+ We probably do not want to use the .yaml format (since it's implementation detail which is subject to change.)</p>

Reply:
agreed, dropping this line.

Reply:
I'm removing the yaml as well. let me know if you think it is worth keeping the yaml file.